producer-c/producer-cloudwatch-integ/jobs/runner.groovy

```diff
@@ -0,0 +1,216 @@
+import jenkins.model.*
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+
+START_TIMESTAMP = new Date().getTime()
+HAS_ERROR = false
+
+RUNNING_NODES=0
+
+CREDENTIALS = [
+    [
+        $class: 'AmazonWebServicesCredentialsBinding', 
+        accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+        credentialsId: 'SDK_CANARY_CREDS',
+        secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
+    ]
+]
+
+def buildProducer() {
+  sh  """
+    cd $WORKSPACE/producer-c/producer-cloudwatch-integ && 
+    mkdir -p build
+    cd build && 
+    sudo cmake .. && 
+    sudo make 
+  """
+}
+
+def buildConsumer(envs) {
+  withEnv(envs) {
+      sh '''
```

Comment:
Why not calling the build-consumer.sh script?

Reply:
It is `run-consumer.sh`. I am adding it as a convenience to run locally. Here I am splitting the steps between `buildConsumer()` and `runConsumer().

Reply:
Oops sorry. I meant to delete this comment before posting it 😅